os/include/arpa/inet.h

```diff
@@ -223,6 +241,9 @@ in_addr_t   inet_network(FAR const char *cp);
 /**
  * @brief Convert the Internet host address to a string in IPv4 dotted-decimal notation.
  *
+ * @details @b #include <arpa/inet.h>\n
+ * SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised